docs/pages/push-notifications/sending-notifications-custom.mdx

```diff
@@ -122,6 +122,18 @@ Your FCM server key can be found by making sure you've followed the [configurati
 
 Communicating with APNs is a little more complicated than with FCM. Some libraries wrap all of this functionality into one or two function calls such as [`node-apn`](https://github.com/node-apn/node-apn). However, in the examples below, a minimum set of libraries are used.
 
+### Client APNs entitlement
+
+The steps below will only work if your iOS app has the APNs entitlement. There are two ways to ensure that this entitlement is added to your app:
+
+- If the app includes the `expo-notifications` package, then the entitlement is added automatically in the Expo configuration.
+- If you are not using the `expo-notifications` package, then you should [add the `apns-environment` entitlement manually to the Expo configuration](/build-reference/ios-capabilities/#entitlements).
+
+For apps using [CNG](/workflow/continuous-native-generation/), the prebuild step will automatically add the entitlement to the generated native iOS files.
```

Comment:
Same as my first comment, this doesn't seem to be true? I mean - expo-notifications config plugin needs to be present in app config for this to happen? 🤔

Reply:
I’ll reword this so that the idea is more clear.